js/view.js

```diff
@@ -32,4 +35,15 @@ projectView.setTeasers = function() {
 $(document).ready(function() {
   projectView.setTeasers();
   projectView.handleMainNav();
+
+  $(window).on('resize', function() {
+    if (window.innerWidth > 640) {
+      $('.menu').show();
+      $('.menu-item').css('color','black');
```

Comment:
You should generally be setting styles like this in CSS using `@media` queries rather than using jQuery. This actually creates a bug in your mobile view menu.